pom.xml

```diff
@@ -9,7 +9,7 @@
   <description>Maven plugin for the automated building of Axon Ivy Projects. Referenced from the pom.xml of Axon Ivy Projects.</description>
   <url>https://github.com/axonivy/project-build-plugin</url>
   <prerequisites>
-    <maven>3.1</maven>
+    <maven>${maven.version}</maven>
```

Comment:
this property will be prominently stated as depndency on the official doc -> https://axonivy.github.io/project-build-plugin/snapshot/9.4/plugin-info.html#System_Requirements